docs/source/en/tutorials/using_peft_for_inference.md

```diff
@@ -12,19 +12,18 @@ specific language governing permissions and limitations under the License.
 
 [[open-in-colab]]
 
-# Inference with PEFT
+# Loading adapters (LoRAs) on diffusers with PEFT for inference
```

Comment:
Just a minor suggestion, but I prefer shorter titles because they're easier to reference and more direct. In this case, what do you think about just removing "adapters" entirely because as you mentioned, users probably aren't looking for "adapters" and the only "adapter" we're showing here anyway is LoRA.

I also think its probably not important to mention PEFT in the title because I don't think users really care how the LoRAs are loaded, just as long as they are :)

```suggestion
# Load LoRAs for inference
```